Mathlib/GroupTheory/Perm/Sign.lean

```diff
@@ -340,7 +340,7 @@ theorem signAux3_mul_and_swap [Finite α] (f g : Perm α) (s : Multiset α) (hs
     signAux3 (f * g) hs = signAux3 f hs * signAux3 g hs ∧
       Pairwise fun x y => signAux3 (swap x y) hs = -1 := by
   obtain ⟨n, ⟨e⟩⟩ := Finite.exists_equiv_fin α
-  obtain ⟨l, rfl⟩ := Quotient.exists_rep s
+  induction s using Quotient.inductionOn with | _ l => ?_
```

Comment:
Perhaps this is controversial, because the new spelling is longer and has more `_`s?